backend/src/db/seed.js

```diff
@@ -931,13 +931,34 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
     const additionalUsers = await Promise.all(
       [...Array(30).keys()].map(() => Factory.build('user')),
     )
+
     await Promise.all(
       additionalUsers.map(async (user) => {
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-05-13T17:47:30Z" title="Wednesday, May 13th 2020, 7:47:30 pm +02:00">May 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -928,16 +928,25 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
       trollingComment.update({ disabled: true, updatedAt: new Date().toISOString(), closed: true }),
     ])
 
-    const additionalUsers = await Promise.all(
-      [...Array(30).keys()].map(() => Factory.build('user')),
+    await Promise.all([...Array(30).keys()].map(() => Factory.build('user')))
+    await Promise.all(
+      [...Array(30).keys()].map((index) => Factory.build('user', { name: `Jenny${index}` })),
     )
     await Promise.all(
-      additionalUsers.map(async (user) => {
```


Is there any reason to eliminate the relations?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-05T10:35:11Z" title="Friday, June 5th 2020, 12:35:11 pm +02:00">Jun 5, 2020</time>_

---


You talking about the `following` relation @Mogge ?
But here nothing has changed. Or I’m mistaken?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-05T10:38:40Z" title="Friday, June 5th 2020, 12:38:40 pm +02:00">Jun 5, 2020</time>_

---


Okay it seems like this is an old comment and the relations were removed in an earlier commit GitHub blame told me.